backend/mora/service/configuration_options.py

```diff
@@ -0,0 +1,167 @@
+#
+# Copyright (c) 2017-2018, Magenta ApS
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+
+'''
+Organisational units
+--------------------
+
+This section describes how to interact with organisational units.
+
+For more information regarding reading relations involving organisational
+units, refer to :http:get:`/service/(any:type)/(uuid:id)/details/`
+
+'''
+import sqlite3
```

Comment:
You've now added a hard requirement that _all_ the WSGI processes share a filesystem, and that they can write to said file system.

Why not just use PostgreSQL, which we already have available and use for authentication? This is an advantage to using SQLAlchemy, BTW; you can easily support multiple types of databases.